2018-03-26 00:33:05,751 ERROR: Exception on /reset_password_request [POST] [in /home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask_mail.py", line 492, in send
    message.send(connection)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask_mail.py", line 427, in send
    connection.send(self)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask_mail.py", line 192, in send
    message.rcpt_options)
  File "/usr/lib/python3.5/smtplib.py", line 847, in sendmail
    self.ehlo_or_helo_if_needed()
  File "/usr/lib/python3.5/smtplib.py", line 599, in ehlo_or_helo_if_needed
    if not (200 <= self.ehlo()[0] <= 299):
  File "/usr/lib/python3.5/smtplib.py", line 439, in ehlo
    self.putcmd(self.ehlo_msg, name or self.local_hostname)
  File "/usr/lib/python3.5/smtplib.py", line 366, in putcmd
    self.send(str)
  File "/usr/lib/python3.5/smtplib.py", line 358, in send
    raise SMTPServerDisconnected('please run connect() first')
smtplib.SMTPServerDisconnected: please run connect() first

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ronnie/coding/microblog/app/routes.py", line 155, in reset_password_request
    send_password_reset_email(user)
  File "/home/ronnie/coding/microblog/app/email.py", line 20, in send_password_reset_email
    user=user, token=token))
  File "/home/ronnie/coding/microblog/app/email.py", line 10, in send_email
    mail.send(msg)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask_mail.py", line 492, in send
    message.send(connection)
  File "/home/ronnie/.local/share/virtualenvs/microblog-_cc4o7Cx/lib/python3.5/site-packages/flask_mail.py", line 152, in __exit__
    self.host.quit()
  File "/usr/lib/python3.5/smtplib.py", line 978, in quit
    res = self.docmd("quit")
  File "/usr/lib/python3.5/smtplib.py", line 419, in docmd
    self.putcmd(cmd, args)
  File "/usr/lib/python3.5/smtplib.py", line 366, in putcmd
    self.send(str)
  File "/usr/lib/python3.5/smtplib.py", line 358, in send
    raise SMTPServerDisconnected('please run connect() first')
smtplib.SMTPServerDisconnected: please run connect() first
2018-03-26 01:37:38,214 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 01:44:23,279 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:19:57,847 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:24:59,973 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:26:33,761 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:29:37,415 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:34:56,600 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:36:27,077 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:47]
2018-03-26 13:51:39,394 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:49]
2018-03-27 02:45:20,730 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:53]
2018-03-27 02:54:31,819 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:53]
2018-03-27 02:58:06,240 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:53]
2018-03-27 03:04:36,860 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:25:49,898 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:31:04,720 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:31:12,123 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:31:33,096 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:31:51,259 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:31:58,229 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:34:42,869 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:34:46,787 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
2018-03-27 03:34:50,468 INFO: Microblog startup [in /home/ronnie/coding/microblog/app/__init__.py:52]
